src/components/stepper/step-label/step-label.stories.ts

```diff
@@ -0,0 +1,70 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { InputType } from '@storybook/types';
+import type { Args, ArgTypes, Decorator, Meta, StoryObj } from '@storybook/web-components';
+import { html, type TemplateResult } from 'lit';
+
+import { sbbSpread } from '../../core/dom';
+
+import readme from './readme.md?raw';
+import './step-label';
+
+const iconName: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const disabled: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const defaultArgTypes: ArgTypes = {
+  disabled,
+  'icon-name': iconName,
+};
+
+const defaultArgs: Args = {
+  disabled: false,
+  'icon-name': 'tick-small',
+};
+
+const Template = (args: Args): TemplateResult =>
+  html`<sbb-step-label ${sbbSpread(args)}>Label</sbb-step-label>`;
+
+export const Default: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs },
+};
+
+export const Selected: StoryObj = {
```

Comment:
Add a story with icon-name set

Reply:
Actually all the stories have an icon-name set, also because otherwise the circle would be empty as the css counter is handled by the stepper component.